pkg/smartcontract/manifest/standard/nep30.go

```diff
@@ -0,0 +1,24 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep30 is a NEP-30 Standard describing smart contract verify method functionality.
+var Nep30 = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name:       "verify",
+					Safe:       true,
+					ReturnType: smartcontract.BoolType,
+				},
+				// Parameters are nil for verify, because NEP-30 allows an undefined number of
+				// parameters for this method. This tells the checkMethod that the exact parameter
+				// count is unspecified and must not be strictly enforced.
```

Comment:
```suggestion
				// Parameters are nil for verify, because NEP-30 allows an undefined number of
				// parameters for this method. This tells the checkMethod that the exact parameter
				// count is unspecified and must not be strictly enforced.
				},
```

This comment is a part of method declaration. And it may be shortened to something like `nil parameters implies no parameters check, NEP-30 allows an undefined number of parameters for verify.` to match comments from the previous commit.